### PR TITLE
i18n(zh-cn): Update `view-transitons.mdx`

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update `view transitons.mdx`

#### Related issues & labels (optional)

- #7250

### CODE REVIEW

**Comment by liruifengv** [comment]
There has a new commit: https://github.com/withastro/docs/commit/f058a76e01936fdf3b863e48e397b71d1306eefc

**Comment by huyikai** [comment]
> There has a new commit: [f058a76](https://github.com/withastro/docs/commit/f058a76e01936fdf3b863e48e397b71d1306eefc)

已更新

File: src/content/docs/zh-cn/guides/view-transitions.mdx [hunk 1]
```diff
@@ -441,21 +454,43 @@ Astro 不会模拟 `initial` 浏览器动画。因此，任何使用此动画的
...
+
+要强制内联脚本在每次过渡后重新执行，请添加 `data-astro-rerun` 属性。向脚本添加任何属性也会隐式地添加 `is:inline`，因此这只适用于未被 Astro 打包和处理的脚本。
+
+```astro
+<script is:inline data-astro-rerun>...</script>
+```
+
+为了确保脚本在客户端导航期间每次加载页面时都能运行，它应该由一个[生命周期事件](#生命周期事件)执行。例如，`DOMContentLoaded` 的事件监听器可以被 [`astro:page-load`](/zh-cn/guides/view-transitions/#astropage-load) 生命周期事件替换。
+
+如果你在内联脚本中设置了全局状态的代码，那么这个状态需要考虑到脚本可能会被执行多次的情况。在你的 `<script>` 标签中检查全局状态，并在可能的情况下有条件地执行你的代码。这是因为 `window` 对象是被保留的。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/view-transitions.mdx:484]:
```suggestion
如果你有在内联脚本中设置了全局状态的代码，那么这个状态需要考虑到脚本可能会被执行多次的情况。在你的 `<script>` 标签中检查全局状态，并在可能的情况下有条件地执行你的代码。这是因为 `window` 对象是被保留的。
```

**Review by liruifengv — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**